webgpu.h

```diff
@@ -21,1002 +21,1004 @@
-    WGPU_BUFFER_USAGE_BIT_INDIRECT = 0x00000100,
-    WGPU_BUFFER_USAGE_BIT_FORCE32 = 0x7FFFFFFF
-} wgpuBufferUsageBit;
+    WGPUBufferUsageBit_None = 0x00000000,
+    WGPUBufferUsageBit_MapRead = 0x00000001,
+    WGPUBufferUsageBit_MapWrite = 0x00000002,
+    WGPUBufferUsageBit_CopySrc = 0x00000004,
+    WGPUBufferUsageBit_CopyDst = 0x00000008,
+    WGPUBufferUsageBit_Index = 0x00000010,
+    WGPUBufferUsageBit_Vertex = 0x00000020,
+    WGPUBufferUsageBit_Uniform = 0x00000040,
+    WGPUBufferUsageBit_Storage = 0x00000080,
+    WGPUBufferUsageBit_Indirect = 0x00000100,
+    WGPUBufferUsageBit_Force32 = 0x7FFFFFFF
+} WGPUBufferUsageBit;
 
 typedef enum {
-    WGPU_COLOR_WRITE_MASK_NONE = 0x00000000,
-    WGPU_COLOR_WRITE_MASK_RED = 0x00000001,
-    WGPU_COLOR_WRITE_MASK_GREEN = 0x00000002,
-    WGPU_COLOR_WRITE_MASK_BLUE = 0x00000004,
-    WGPU_COLOR_WRITE_MASK_ALPHA = 0x00000008,
-    WGPU_COLOR_WRITE_MASK_ALL = 0x0000000F,
-    WGPU_COLOR_WRITE_MASK_FORCE32 = 0x7FFFFFFF
-} wgpuColorWriteMask;
+    WGPUColorWriteMask_None = 0x00000000,
+    WGPUColorWriteMask_Red = 0x00000001,
+    WGPUColorWriteMask_Green = 0x00000002,
+    WGPUColorWriteMask_Blue = 0x00000004,
+    WGPUColorWriteMask_Alpha = 0x00000008,
+    WGPUColorWriteMask_All = 0x0000000F,
+    WGPUColorWriteMask_Force32 = 0x7FFFFFFF
+} WGPUColorWriteMask;
 
 typedef enum {
-    WGPU_FACE_NONE = 0x00000000,
-    WGPU_FACE_BACK = 0x00000001,
-    WGPU_FACE_FRONT = 0x00000002,
-    WGPU_FACE_BOTH = 0x00000003,
-    WGPU_FACE_FORCE32 = 0x7FFFFFFF
-} wgpuFace;
+    WGPUFace_None = 0x00000000,
+    WGPUFace_Back = 0x00000001,
+    WGPUFace_Front = 0x00000002,
+    WGPUFace_Both = 0x00000003,
+    WGPUFace_Force32 = 0x7FFFFFFF
+} WGPUFace;
 
 typedef enum {
-    WGPU_SHADER_STAGE_BIT_NONE = 0x00000000,
-    WGPU_SHADER_STAGE_BIT_VERTEX = 0x00000001,
-    WGPU_SHADER_STAGE_BIT_FRAGMENT = 0x00000002,
-    WGPU_SHADER_STAGE_BIT_COMPUTE = 0x00000004,
-    WGPU_SHADER_STAGE_BIT_FORCE32 = 0x7FFFFFFF
-} wgpuShaderStageBit;
+    WGPUShaderStageBit_None = 0x00000000,
+    WGPUShaderStageBit_Vertex = 0x00000001,
+    WGPUShaderStageBit_Fragment = 0x00000002,
+    WGPUShaderStageBit_Compute = 0x00000004,
+    WGPUShaderStageBit_Force32 = 0x7FFFFFFF
+} WGPUShaderStageBit;
 
 typedef enum {
-    WGPU_TEXTURE_USAGE_BIT_NONE = 0x00000000,
-    WGPU_TEXTURE_USAGE_BIT_COPY_SRC = 0x00000001,
-    WGPU_TEXTURE_USAGE_BIT_COPY_DST = 0x00000002,
-    WGPU_TEXTURE_USAGE_BIT_SAMPLED = 0x00000004,
-    WGPU_TEXTURE_USAGE_BIT_STORAGE = 0x00000008,
-    WGPU_TEXTURE_USAGE_BIT_OUTPUT_ATTACHMENT = 0x00000010,
-    WGPU_TEXTURE_USAGE_BIT_FORCE32 = 0x7FFFFFFF
-} wgpuTextureUsageBit;
+    WGPUTextureUsageBit_None = 0x00000000,
+    WGPUTextureUsageBit_CopySrc = 0x00000001,
+    WGPUTextureUsageBit_CopyDst = 0x00000002,
+    WGPUTextureUsageBit_Sampled = 0x00000004,
+    WGPUTextureUsageBit_Storage = 0x00000008,
+    WGPUTextureUsageBit_OutputAttachment = 0x00000010,
+    WGPUTextureUsageBit_Force32 = 0x7FFFFFFF
```

Comment:
this is not a bit :)

Reply:
Will be fixed when the flags match what you described :)